cmd/root.go

```diff
@@ -36,6 +36,7 @@ var rootCmd = &cobra.Command{
 	Long: `DAG Image Builder helps building a complex image dependency graph
 
 Run dib --help for more information`,
+	SilenceUsage: true,
```

Comment:
I think `SilenceUsage: true` here is sufficient since the other commands inherit from root

Reply:
Yes, the intention of `SilenceUsage: true` in the child commands is more for documentation purposes, even though it is already enforced by the parent. What do you think?

Reply:
sounds reasonable, ok for me